docs/methods/authorizeCoinJoin.md

```diff
@@ -0,0 +1,61 @@
+## Bitcoin: authorize CoinJoin
+Allow device to do preauthorized operations in `signTransaction`, `getOwnershipProof` and `getOwnershipId` methods without further user interaction.
+
+Permission persists until device disconnection or `maxTotalFee` limit is reached.
+
+ES6
+```javascript
+const result = await TrezorConnect.authorizeCoinJoin(params);
+```
+
+CommonJS
+```javascript
+TrezorConnect.authorizeCoinJoin(params).then(function(result) {
+
+});
+```
+
+> :warning: **This feature is experimental! Do not use it in production!**
+
+> :warning: **Supported only by Trezor T with Firmware 2.4.4 or higher!**
+
+### Params
+[****Optional common params****](commonParams.md)
+#### Exporting single id
+* `path` — *required* `string | Array<number>` minimum length is `3`. [read more](path.md)
+* `maxTotalFee` — *required* `number`
+* `feePerAnonymity` — *optional* `number`
```

Comment:
Note that due to the different fee model in WabiSabi, I will be renaming `feePerAnonymity` to `maxFeeRate` in the protobuf definition and it will be in integer in units of 10^-8 percent.